sfdx-project.json

```diff
@@ -7,5 +7,5 @@
   ],
   "namespace": "",
   "sfdcLoginUrl": "https://login.salesforce.com",
-  "sourceApiVersion": "50.0"
+  "sourceApiVersion": "51.0"
```

Comment:
@scolladon maybe we can also upgrade the API version of the other metadata files, like the components? What do you think?
Suggestion: we can use this plugin https://github.com/muenzpraeger/sfdx-plugin with the command `sfdx mzpr:source:api:set`... and let the magic happens ! 😄

Reply:
That is a very good idea !
I add it in this PR